pkg/utils/kubeUtils.go

```diff
@@ -34,6 +34,21 @@ func DoHelmUpgrade(project string, stage string) error {
 	return err
 }
 
+// RestartPodsWithSelector restarts the pods which are found in the provided namespace and selector
+func RestartPodsWithSelector(useInClusterConfig bool, namespace string, selector string) error {
+	clientset, err := GetKubeAPI(useInClusterConfig)
+	if err != nil {
+		return err
+	}
+	pods, err := clientset.Pods(namespace).List(metav1.ListOptions{LabelSelector: selector})
```

Comment:
let's also check if err != nil after this call

Reply:
Thanks!